src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -269,10 +269,10 @@ private void updatePrivilegesInSessionLevel(String sectionName, String sessionNa
     }
 
     private void verifyExistenceOfsectionName(String sectionName) {
-        if (!this.sessionLevel.containsKey(sectionName)) {
+        this.sessionLevel.computeIfAbsent(sectionName, key -> {
             addSectionWithDefaultPrivileges(sectionName);
-            this.sessionLevel.put(sectionName, new LinkedHashMap<String, Map<String, Boolean>>());
-        }
+            return new LinkedHashMap<String, Map<String, Boolean>>();
```

Comment:
you can simply return `new LinkedHashMap<>`

Reply:
I decided to return `new LinkedHashMap<String, Map<String, Boolean>>` insted of simply `new LinkedHashMap` in order to try to not change the old code. In the old code we were putting a `new LinkedhashMap<String, Map<String, Boolean>>` in the map `this.sessionLevel` so I decided to do the same.


**Old Code**
```java
    private void verifyExistenceOfsectionName(String sectionName) {
        if (!this.sessionLevel.containsKey(sectionName)) {
            addSectionWithDefaultPrivileges(sectionName);
            this.sessionLevel.put(sectionName, new LinkedHashMap<String, Map<String, Boolean>>());
        }
    }
```
**New Code**
```java
    private void verifyExistenceOfsectionName(String sectionName) {
        this.sessionLevel.computeIfAbsent(sectionName, key -> {
            addSectionWithDefaultPrivileges(sectionName);
            return new LinkedHashMap<String, Map<String, Boolean>>();
        });
    }
```

Reply:
Yeah I know, but that part is unnecessary even in the old code. Not all parts of the old code e correct :P

Reply:
Hi @pedrolarben you r right, they r equivalent. We shall change it to make the code cleaner :)